app/assets/stylesheets/welcome.scss.erb

```diff
@@ -354,3 +356,22 @@ nav.custom_nav{
 a.remove-text {
   color: #ffffff !important;
 }
+
+ a.brand-logo.logo-padding { 
+   padding-left: 20px;
+}
+
+.divider {
+  background-color: rgba(0, 128, 128, 0.6);
+  width: 100%;
+  height: 120px;
+  margin: 0px 0px;
+}
+
+#search-events {
```

Comment:
Perhaps you would want to remove CSS selectors that have no rules in them. Like this one